src/CyPhyMasterInterpreter/Rules/StructuralFEATestBenchChecker.cs

```diff
@@ -56,7 +56,7 @@ protected IEnumerable<ContextCheckerResult> FaceConstructMustBeAbaqusModelBased(
                         {
                             var feedback = new ContextCheckerResult()
                             {
-                                Success = false,
+                                Success = true,

```

Comment:
Changing this to "Success = true" doesn't line up with the Message. Should this ContextCheckerResult be removed? i.e. is the Face construct allowed for ABAQUS deck based now?